### PR TITLE
UseInferredMemberNamed codefix show treat spaces as elastic

### PR DESCRIPTION
**Pull request by jcouv**
**Customer scenario**

With language version 7.1, use the "simplify tuple name" feature to simplify the "a" member below:
```C#
            int a = 1;
            var t = (a: a, 2); // becomes `var t = ( a, 2);` with extra space before `a`
```

**Bugs this fixes:**
Fixes https://github.com/dotnet/roslyn/issues/22347

**Workarounds, if any**
Format selection or document.

**Risk**
**Performance impact**
Low. This PR only affects this one refactoring.

**Is this a regression from a previous update?**
No.

**How was the bug found?**
Reported by customers.

@sharwell @rchande @dotnet/roslyn-ide for review. Also, please advise whether this should be considered for 15.5 or moved out to 15.later. Thanks

### CODE REVIEW

File: src/Features/VisualBasic/Portable/UseInferredMemberName/VisualBasicUseInferredMemberNameCodeFixProvider.vb [hunk 1]
```diff
@@ -33,7 +33,7 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.UseInferredMemberName
                 Dim node = root.FindNode(diagnostic.Location.SourceSpan)
                 Select Case node.Kind
                     Case SyntaxKind.NameColonEquals
-                        editor.RemoveNode(node, SyntaxRemoveOptions.KeepExteriorTrivia)
+                        editor.RemoveNode(node, SyntaxRemoveOptions.KeepExteriorTrivia Or SyntaxRemoveOptions.AddElasticMarker)
                         Exit Select
 
                     Case SyntaxKind.NamedFieldInitializer
```
Comment (CyrusNajmabadi) [inline src/Features/VisualBasic/Portable/UseInferredMemberName/VisualBasicUseInferredMemberNameCodeFixProvider.vb]: is there a similar issue for NamedFieldInitializer?

Comment (jcouv) [reply src/Features/VisualBasic/Portable/UseInferredMemberName/VisualBasicUseInferredMemberNameCodeFixProvider.vb]: Not as far as I can tell. I have spaces in VB object initializers in various positions and also tested a couple more, but I did not notice any misbehavior.

**Comment by jcouv** [comment]
@jinujoseph What do you think about this for 15.5? I don't know where the IDE bar currently stands.

File: src/EditorFeatures/CSharpTest/UseInferredMemberName/UseInferredMemberNameTests.cs [hunk 2]
```diff
@@ -110,7 +110,7 @@ class C
     void M()
     {
         int a = 1;
-        var t = new { [||]a=a, 2 };
+        var t = new { [||]a= a, 2 };
```
Comment (jasonmalinowski) [inline src/EditorFeatures/CSharpTest/UseInferredMemberName/UseInferredMemberNameTests.cs]: Why the change here?

Comment (jcouv) [reply src/EditorFeatures/CSharpTest/UseInferredMemberName/UseInferredMemberNameTests.cs]: Extra trivia in the input should be considered elastic, so the output should not be affected in this case.

**Comment by jcouv** [comment]
Ping @jinujoseph for milestone question.

**Comment by jinujoseph** [comment]
This won't meet the 15.5 escrow bar , can you retarget the PR to "post-dev15.5-contrib" branch ? so we can take this for 15.6?

**Comment by jcouv** [comment]
I think `post-dev15.5-contrib` is the next foundational version. I'll hold-off until we hear more details on proper branch for `15.6`.
Thanks

**Comment by jcouv** [comment]
test ubuntu_14_debug_prtest please

**Comment by jcouv** [comment]
test ubuntu_16_debug_prtest please
